app/controllers/ctc/questions/dependents/child_can_be_claimed_by_other_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Ctc
+  module Questions
+    module Dependents
+      class ChildCanBeClaimedByOtherController < BaseDependentController
+        include AuthenticatedCtcClientConcern
+        layout "yes_no_question"
+
+        def self.show?(dependent)
+          return false unless dependent
```

Comment:
Trivial: Rubocop and Asheesh would suggest a blank line after a guard like this.

Not a blocker.